### PR TITLE
Replace Gitter link with forum url

### PR DESCRIPTION
**Pull request by topokel**

### CODE REVIEW

File: src/config.toml [hunk 1]
```diff
@@ -18,5 +18,5 @@ github_url_master_examples = "https://github.com/amethyst/amethyst/tree/master/e
 github_url_base = "https://github.com/amethyst/amethyst"
 reddit_url = "https://reddit.com/r/amethyst"
 discord_invite = "https://discord.gg/E2Dd244"
-gitter_url = "https://gitter.im/amethyst/general"
+forum_url = "http://community.amethyst-engine.org/"
```
Comment (NaakkaDev) [inline src/config.toml]: This should be `https`. ;)

**Review by NaakkaDev — CHANGES_REQUESTED** [review]
Seems good aside from the http protocol being used on the community url.

**Review by Moxinilian — APPROVED** [review]
As soon as the previous conversation is resolved.

File: src/config.toml (hunk 1 above)
Comment (topokel) [reply src/config.toml]: fixed ty

**Review by NaakkaDev — APPROVED** [review]

**Review by happenslol — APPROVED** [review]
Looks good, thanks!

**Comment by LucioFranco** [comment]
thanks 👍 

bors r+

**Comment by LucioFranco** [comment]
ignore me need coffee 

bors r+

**Pull request merged**